tests/test.ts

```diff
@@ -188,7 +228,15 @@ test('is.buffer', t => {
 });
 
 test('is.object', t => {
-	for (const el of types.get('object')) {
+	const testData = types.get('object');
+
+	if (testData === undefined) {
+		t.fail('is.object not defined');
+
```

Comment:
remove this empty line

Reply:
The linter config insists on that empty line before return, sry 👍